c3pzero_ignition/CMakeLists.txt

```
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 3.22)
+project(c3pzero_ignition CXX)
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
```

```suggestion
  add_compile_options(-Werror -Wall -Wextra -Wpedantic)
```
What's the point of warnings if you never have to fix them?

---

👍🏻